### DISCUSSION TITLE
"Unknown subscription" error

### DISCUSSION THREAD

**Discussion by shijir0927** [anycable/anycable#219] [Q&A]
Hi,

I noticed I am frequently getting this error in my logs for my anycable server (1.4.1). I am using Speedy Streams and Reliable Streams. I'm not sure what this error exactly means and it seems to be happening a every few minutes. Would be great to know where this error message is coming from and can someone help us debug and fix this error? Thank you.

`Failed to handle incoming message '{"identifier":"{\"channel\":\"Turbo::StreamsChannel\",\"signed_stream_name\":\"IloybGtPaTh2WW1GamEyVnlhMmwwTDBOeWIzZGtablZ1WkdsdVp6bzZVSEp2YW1WamRDOHhPQSI=--315b9740c974ded215944bcbea406efef0f6a37833d17de463370daf6e6c95a5\"}","command":"history","history":{"since":1698304211,"streams":{}}}' with error: Unknown subscription {"channel":"Turbo::StreamsChannel","signed_stream_name":"IloybGtPaTh2WW1GamEyVnlhMmwwTDBOeWIzZGtablZ1WkdsdVp6bzZVSEp2YW1WamRDOHhPQSI=--315b9740c974ded215944bcbea406efef0f6a37833d17de463370daf6e6c95a5"}`

**Comment by palkan** [comment]
Hey,

Do you use `anycable-client`? It would be helpful if you can share its debug logs (pass `logLevel: 'debug'` to `createCable` or `createConsumer`); I think, it happens during re-connects; looks like a client requests history for the already unsubscribed channel
